contracts/script/Deploy.s.sol

```diff
@@ -11,56 +11,67 @@ contract DeployScript is Script {
     function run() public {
         vm.startBroadcast();
 
-        bytes32 headerRangeFunctionId = vm.envBytes32(
-            "HEADER_RANGE_FUNCTION_ID"
-        );
-        bytes32 rotateFunctionId = vm.envBytes32("ROTATE_FUNCTION_ID");
-        uint32 height = uint32(vm.envUint("GENESIS_HEIGHT"));
-        bytes32 header = vm.envBytes32("GENESIS_HEADER");
-        uint64 authoritySetId = uint64(vm.envUint("GENESIS_AUTHORITY_SET_ID"));
-        bytes32 authoritySetHash = vm.envBytes32("GENESIS_AUTHORITY_SET_HASH");
+        bytes32 create2Salt = bytes32(vm.envBytes("CREATE2_SALT"));
 
-        address gateway = vm.envAddress("GATEWAY_ADDRESS");
-
-        bytes32 CREATE2_SALT = bytes32(vm.envBytes("CREATE2_SALT"));
-
-        bool UPGRADE = vm.envBool("UPGRADE_VIA_EOA");
-        address existingProxyAddress = vm.envAddress("CONTRACT_ADDRESS");
+        bool upgrade = vm.envBool("UPGRADE");
 
         // Deploy contract
-        VectorX lightClientImpl = new VectorX{salt: bytes32(CREATE2_SALT)}();
+        VectorX lightClientImpl = new VectorX{salt: bytes32(create2Salt)}();
 
         console.logAddress(address(lightClientImpl));
 
         VectorX lightClient;
-        if (!UPGRADE) {
+        if (!upgrade) {
             lightClient = VectorX(
                 address(
-                    new ERC1967Proxy{salt: bytes32(CREATE2_SALT)}(
+                    new ERC1967Proxy{salt: bytes32(create2Salt)}(
                         address(lightClientImpl),
                         ""
                     )
                 )
             );
+
+            // Initialize the Vector X light client.
+            lightClient.initialize(
+                VectorX.InitParameters({
+                    // TODO: Migrate to using upgrade scripts in SuccinctX that work with Gnosis Safe.
```

Comment:
the issue is that this gnosis safe is not going to be managed through the same infra since it'll have avail people on it, so i don't think we'll ever be able to do that